roles/fio-devel/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Stephen Bates, 2022
+#
+# Clone the upstream fio repo from Jens and checkout the SHA given by
+# the associated input variable. Also ensure the packages required to
+# run specific tests are included and also update this version of fio
+# to the PATH.
+
+- name: Update package cache (assumes apt)
+  ansible.builtin.apt:
+    update_cache: yes
+  become: yes
+
+- name: Install fio related favourite packages
+  ansible.builtin.package:
+    name:
+      - gettext
+      - libaio-dev
+      - libnuma-dev
+      - libpci-dev
+      - libpmem-dev
+      - libpmem2-dev
+      - liburing-dev
+      - linux-tools-common
+      - nvme-cli
+      - pkg-config
+  become: yes
+
+- name: Install fio related favourite packages when on AWS
+  ansible.builtin.package:
+    name:
+      - linux-tools-aws
+  become: yes
+  when: inventory_hostname in groups['awsmachines']
+
+- name: Checkout upstream fio repo from Github
+  ansible.builtin.git:
+    repo: https://github.com/axboe/fio.git
+    dest: ~/fio
+    force: yes
+    version: '{{ fio_tag }}'
+
+- name: Configure fio and record output of the configuration
+  ansible.builtin.shell:
+    chdir: ~/fio
+    cmd: ./configure --prefix='{{ fio_install_dir }}' | tee config.tee
+
+- name: Make fio
+  ansible.builtin.shell:
```

Comment:
You can use the [make module](https://docs.ansible.com/ansible/latest/collections/community/general/make_module.html)

Reply:
I did not like that module and prefer using the shell.